src/test/java/DeadlineTest.java

```diff
@@ -0,0 +1,17 @@
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DeadlineTest {
+    @Test
+    public void toStringTest(){
+        Deadlines  deadline = new Deadlines("return book", "2/12/2019 1800", false);
+        assertEquals("[D][ ] return book (by: Dec 2 2019 6:00 PM)", deadline.toString());
+    }
+
+    @Test
+    public void toSaveFormatTest(){
+        Deadlines  deadline = new Deadlines("return book", "2/12/2019 1800", false);
```

Comment:
Should take note to avoid adding additional/unnecessary whitespaces in your code, such as the double space in "Deadlines  deadline = "